components/Blog/blogList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+const BlogComponent = ({ content }) =>
```

Comment:
`BlogComponent` is a non-descriptive name. What is this component? What is its purpose? Please update the name of the component to be explicit about its intention.

Reply:
Now I changed `BlogComponent` to just `Blog`...  Even now it doesn't sound good... but better than previous I hope :)